CHANGELOG.md

```diff
@@ -11,6 +11,10 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 -   Minimum `tailwindcss` version required
 
+### Added
+
+-   Implement `Avatar` component
```

Comment:
We should already have an `added` section on the `main` branch with the `InputDate` component

Reply:
Huh, I wonder why it's not conflicting 🤔